client/src/components/SightsList/sightsList.scss

```diff
@@ -0,0 +1,275 @@
+main.country {
+  .sight-slider {
+    position: relative;
+    margin: 13rem auto 4.5rem;
+    & > .wrapper {
+      position: relative;
+    }
+    .subtitle {
+      margin-bottom: 2.2rem;
+      padding-left: 7rem;
+      color: #0d444d;
+      font-weight: 500;
+    }
+
+    .fullScr {
+      position: absolute;
+      top: 0;
+      right: 1rem;
+      padding-left: 7rem;
+      img {
+        width: 3rem;
+        height: 3rem;
+        &:hover {
+          cursor: pointer;
+          transform: scale(1.2);
+        }
+      }
+      &.hide {
+        display: none;
+      }
+    }
+  }
+}
+
+.wrapper:fullscreen {
+  .slider {
+    height: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    flex-flow: wrap;
+    justify-content: space-evenly;
+    align-content: space-evenly;
+
+    .slide {
+      pointer-events: none;
+
+      &:hover {
+        cursor: none;
+      }
+    }
+  }
+}
+
+.slider {
+  .slide {
+    position: relative;
+    height: 24rem;
+    width: 41.9rem;
+    border-radius: 1.75rem;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    cursor: pointer;
+    &:focus {
+      outline: none;
+    }
+    .overlay {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      right: 0;
+      opacity: 0;
+      background: #000;
+      border-radius: 1.75rem;
+      transition: all 300ms linear;
+    }
+
+    &:hover .overlay {
+      opacity: 0.6;
+      transition: all 300ms linear;
+    }
+    &:hover .slide__title {
+      color: white;
+      transition: all 300ms linear;
+    }
+
+    .slide__title {
+      position: absolute;
+      top: 30%;
+      left: 50%;
+      transform: translate(-50%);
+
+      width: 100%;
+      padding: 1rem;
+
+      font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans",
+        "Droid Sans", "Helvetica Neue", sans-serif;
+      font-style: normal;
+      font-weight: 500;
+      color: transparent;
+      font-size: 3rem;
+      text-align: center;
+      text-transform: uppercase;
+      transition: all 300ms linear;
+    }
+
+    .rating {
+      position: absolute;
+      right: 1.6rem;
+      bottom: 1.6rem;
+      height: 3.5rem;
+      width: 7rem;
+
+      display: flex;
+      flex-direction: row;
+      justify-content: space-evenly;
+      align-items: center;
+
+      background: linear-gradient(91.97deg, #2a8086 14.73%, #0d444c 97.52%);
+      box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.1);
+      border-radius: 4rem;
+
+      font-family: "Roboto", -apple-system, BlinkMacSystemFont, "Segoe UI", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans",
```

Comment:
`font-family` duplicates many times, you may 

create `<Text ` component for the text style and put in there
or create shared scss mixin for this